set.mm

```diff
@@ -727059,7 +727059,7 @@ last digit of the integer part (for ` b = ` 10 the first digit before
        different problems can be shown to be equivalent.  Thus, Conway
        preferred defining surreal numbers as equivalence classes of surreal
        number forms, rather than sign-expansions.  Although sign-expansions are
-       easier to implement in ZF set theory, Conway argued that "formalisation
+       easier to implement in ZF set theory, Conway argued that "formalization
```

Comment:
Is this a citation? Then it should not be modified.

Reply:
He uses "s" so I reverted to that. I also wrote the reference in full (full title of the book, and John "Horton" Conway, so as not to confuse him with the functional analyst).